tests/integ/test_tfs.py

```diff
@@ -46,6 +46,24 @@ def tfs_predictor(instance_type, sagemaker_session, tf_full_version):
         yield predictor
 
 
+@pytest.fixture(scope='module')
+def tfs_predictor_with_accelerator(sagemaker_session, tf_full_version):
+    endpoint_name = sagemaker.utils.unique_name_from_base("sagemaker-tensorflow-serving")
+    instance_type = 'ml.c4.large'
+    accelerator_type = 'ml.eia1.medium'
+    model_data = sagemaker_session.upload_data(path='tests/data/tensorflow-serving-test-model.tar.gz',
+                                               key_prefix='tensorflow-serving/models')
+    with tests.integ.timeout.timeout_and_delete_endpoint_by_name(endpoint_name, sagemaker_session):
+        model = Model(model_data=model_data, role='SageMakerRole',
+                      framework_version=tf_full_version,
+                      sagemaker_session=sagemaker_session,
+                      # passing an existing image for now because ei image is not in prod account yet
+                      image='562200002688.dkr.ecr.us-west-2.amazonaws.com/sagemaker-tensorflow-serving-eia:{}-cpu'
```

Comment:
Before this PR gets merged, this will need to change.

Reply:
I've updated PR description with this information.